designcompose/src/main/java/com/android/designcompose/FrameRender.kt

```diff
@@ -703,10 +703,9 @@ internal fun squooshShapeRender(
             // layout doesn't actually consider rotation yet.
             rectSize,
             customArcAngle,
-            vectorScaleX,
-            vectorScaleY,
-            0, // XXX: layoutId
+            node.layoutId,
```

Comment:
Does this change fix or change anything? Why was it 0 before?

Reply:
It isn't used in Squoosh, and it wasn't easily accessible originally. Rendering uses the layout id to look up if there's a custom size applied to a node, which Squoosh doesn't support. So this doesn't change existing behavior (LayoutManager's custom size hashmap is still empty), but the layout ID is stable enough to use as a key in the (new) shape cache.